lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum.pm

```diff
@@ -301,14 +298,13 @@ sub generate_summarize_mutation_spectrum_result {
 
 sub generate_mutation_rate_result {
     my $self                     = shift;
-    my $somvar_build             = shift;
     my $final_name               = shift;
     my $tier1_snvs               = shift;
     my $tier2_snvs               = shift;
     my $tier3_snvs               = shift;
     my $sub_outdir4              = shift;
     my $data_type                = $self->datatype;
-    my $reference_annotation_dir = $somvar_build->model->annotation_build->data_directory;
+    my $reference_annotation_dir = $self->somatic_build->model->annotation_build->data_directory;
```

Comment:
This is another circumstance where I don't believe delegating to the model is appropriate.